addons/website/views/snippets/s_share.xml

```diff
@@ -2,7 +2,7 @@
 <odoo>
 
 <template id="s_share" name="Share">
-    <div t-attf-class="s_share #{_classes}">
+    <div t-attf-class="s_share text-left #{_classes}">
```

Comment:
Why is this needed ?

Reply:
We now have a new snippet option for s_share to center left/center/right.
Defaulting to left